client/landing/gutenboarding/onboarding-block/site-title.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import React from 'react';
+import { __ as NO__ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '../stores/onboard';
+import { StepInputProps } from './stepper-wizard';
+import Question from './question';
+
+export default function SiteTitle( { onSelect, inputClass, ...props }: StepInputProps ) {
```

Comment:
Let's destructure props explicitly :grimacing: We've had some instances where we did `...props` in Calypso in cases like this (where we actually knew which props to expect, and basically just used it as a shorthand), and it didn't turn out to be good practice :grimacing: 

(Same for the other step components.)